doc/visual-programming/source/index.rst

```diff
@@ -103,9 +110,15 @@ Unsupervised
    widgets/unsupervised/correspondenceanalysis
    widgets/unsupervised/distancemap
    widgets/unsupervised/distances
+   widgets/unsupervised/distancematrix
+   widgets/unsupervised/distancetransformation
+   widgets/unsupervised/distancefile
+   widgets/usnupervised/savedistancematrix
    widgets/unsupervised/hierarchicalclustering
    widgets/unsupervised/kmeansclustering
    widgets/unsupervised/mds
+   widgets/unsupervised/silhouetteplot
```

Comment:
Silhouette is now in Visualize. I know things move around a lot, but this seems to be quite permanent and should switch while we can.